module/Makefile.bsd

```diff
@@ -400,6 +400,18 @@ beforeinstall:
 
 .include <bsd.kmod.mk>
 
+# Generated binary search code is particularly bad with this optimization.
+# Oddly, range_tree.c is not affected when unrolling is not done and dsl_scan.c
+# is not affected when unrolling is done.
+# Disable it until the following upstream issue is resolved:
+# https://github.com/llvm/llvm-project/issues/62790
+.if ${CC} == "clang"
```

Comment:
A minor nit. Clang will ignore this on other architectures, but if that behavior changes, this will break builds. We should wrap this in a check for amd64 or i386.

Reply:
Fixed.